### PR TITLE
IntelliJ Plugin: Load graph in the background

### PR DESCRIPTION
**Pull request by Leland-Takamine**
Load PsiClasses and compute DI graph in the background so we don't block the UI on refresh.

### CODE REVIEW

File: intellij/src/main/kotlin/motif/intellij/MotifPanel.kt [hunk 1]
```diff
@@ -74,9 +77,18 @@ class MotifPanel(
     }
 
     private fun refresh() {
-        ApplicationManager.getApplication().invokeLater {
-            builder.queueUpdate()
-        }
+        ProgressManager.getInstance().run(object : Task.Backgroundable(project, "Refresh") {
```
Comment (tyvsmith) [inline intellij/src/main/kotlin/motif/intellij/MotifPanel.kt]: Automated tests?

File: intellij/src/main/kotlin/motif/intellij/MotifPanel.kt [hunk 2]
```diff
@@ -74,9 +77,18 @@ class MotifPanel(
...
+
+            override fun run(indicator: ProgressIndicator) {
+                ApplicationManager.getApplication().runReadAction {
+                    val graph = graphFactory.compute()
```
Comment (tyvsmith) [inline intellij/src/main/kotlin/motif/intellij/MotifPanel.kt]: How does this handle interrupts? Cancel task, new build from CLI, etc..?

**Comment by Leland-Takamine** [comment]
@tyvsmith Good questions - As this is just a prototype, I'd like to punt these until we are more confident in the approach. I'm going to abandon this PR and open another one against a feature branch so it's clearer that this is not production code (yet).

**Comment by Leland-Takamine** [comment]
#101

**Pull request closed without merging**